tests/unit/test_messages_integration.py

```diff
@@ -230,11 +230,18 @@ def test_progress_brief_terminal(capsys):
     emit.progress("Another message.")
     emit.ended_ok()
 
-    expected = [
+    expected_term = [
         Line("The meaning of life is 42.", permanent=False),
-        Line("Another message.", permanent=True),  # stays as it's the last message
+        Line("Another message.", permanent=False),
+        # This cleaner line is inserted by the printer stop
+        # sequence to reset the last ephemeral print to terminal.
+        Line("", permanent=False),
```

Comment:
Please duplicate this test and have one as before (with permanent last line) and one as you're suggesting (with ephimeral last line). This way we ensure that this will work correctly for ever in BOTH situations.

Reply:
I believe the test ```test_progress_brief_permanent``` covers what you are asking for. Let me know if I made a mistake.

Reply:
No, the other one is about the first (or intermediate) message. But in its way it's testing that, so we're ok.